src/components/step-flow/step-flow.spec.tsx

```diff
@@ -49,60 +53,134 @@ describe("Step Flow component", () => {
   describe("prop checks", () => {
     it("when the 'category' prop is passed, the correct element and text renders", () => {
       render(
-        <StepFlow
-          title="foo"
-          currentStep={5}
-          totalSteps={6}
-          category="bar"
-          titleRef={mockRef}
-        />
+        <StepFlow title="foo" currentStep={5} totalSteps={6} category="bar" />
       );
 
       expect(screen.getByText("bar")).toBeInTheDocument();
     });
 
     it("when the 'title' prop is passed, the correct element and text renders", () => {
       render(
+        <StepFlow title="baz" currentStep={5} totalSteps={6} category="bar" />
+      );
+
+      expect(screen.getByText("baz")).toBeInTheDocument();
+    });
+
+    it("when the 'titleVariant' prop is not passed, the variant is h1 by default", () => {
+      const { container } = render(
+        <StepFlow title="this title is a h1" currentStep={5} totalSteps={6} />
```

Comment:
comment (non-blocking, don't actually change anything): I suppose depending if you say aitch or haitch could affect whether it's `a h1` or `an h1` here 😆